### DISCUSSION TITLE
Changing best-guess article images (like slashdot)

### DISCUSSION THREAD

**Discussion by jsg2021** [FreshRSS/FreshRSS#5415] [Q&A] [answered]
I'm thinking about an extension or maybe a PR that would allow removing the "best guess" thumbnail of an article that does not have one. (per feed)

Does this already exist?

**Comment by math-GH** [comment]
Could you please give a concrete example of a feed where the chosen thumbnail is not a good one?

**Reply by jsg2021** [reply]
On this feed, the image is a tweet button:
http://rss.slashdot.org/Slashdot/slashdot

**Comment by gknops** [comment]
That is bugging me too. Same feed, thumbnail ends up being a super-scaled up section of an old tweet button image. Some way to disable this on a per-feed level would be great. Did anything come of this?

**Reply by Alkarex** [reply, marked as answer]
For that specific feed, you can exclude those thumbnails by adding the following code to the CSS filter to exclude elements:
```
div[data-sanitized-class~='share_submission']
```


Here is the full OPML:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<opml xmlns:frss="https://freshrss.org/opml" version="2.0">
	<head>
		<title>FreshRSS</title>
		<dateCreated>Sun, 04 Jan 2026 20:22:42 +0100</dateCreated>
	</head>
	<body>
		<outline text="Slashdot" type="rss"
			xmlUrl="http://rss.slashdot.org/slashdot/slashdotMain"
			htmlUrl="http://slashdot.org/"
			description="News for nerds, stuff that matters"
			frss:cssContentFilter="div[data-sanitized-class~='share_submission']"
		/>
	</body>
</opml>

```

**Reply by jsg2021** [reply]
awsome! here is the UI place for that filter:

<img width="1509" height="705" alt="Screenshot 2026-01-04 at 4 12 37 PM" src="https://github.com/user-attachments/assets/5be2de38-8b7e-443f-bac7-4fbd5ed3538f" />

**Reply by gknops** [reply]
Awesome, thanks so much for the tip! Works as advertised.
